### PR TITLE
Added Xtend formatter settings

### PR DESCRIPTION
**Pull request by sigiesec**
This change is [Reviewable](https://reviewable.io/reviews/btc-ag/service-idl/102)

### CODE REVIEW

File: com.btc.serviceidl.ide/.settings/org.eclipse.xtend.core.Xtend.prefs [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+blank.lines.before.first.member=0
+blank.lines.between.classes=1
+blank.lines.between.enum.literals=0
+blank.lines.between.fields=0
+blank.lines.between.fields.and.methods=1
+blank.lines.between.imports=0
+blank.lines.between.methods=1
+braces.in.new.line=true
+eclipse.preferences.version=1
+indentation=\t
```
Comment (huttenlocher) [inline com.btc.serviceidl.ide/.settings/org.eclipse.xtend.core.Xtend.prefs]: Does this mean using tabs for indentation? At the moment, all files are formatted using the "insert spaces for tabs" policy.

Comment (sigiesec) [reply com.btc.serviceidl.ide/.settings/org.eclipse.xtend.core.Xtend.prefs]: Good question. 
I don't think that tabs should be used.

However, in the UI I cannot find any setting related to this. I will check that.

Comment (sigiesec) [reply com.btc.serviceidl.ide/.settings/org.eclipse.xtend.core.Xtend.prefs]: Hm, there is an open issue related to this: https://bugs.eclipse.org/bugs/show_bug.cgi?id=405956

Comment (sigiesec) [reply com.btc.serviceidl.ide/.settings/org.eclipse.xtend.core.Xtend.prefs]: indentation.length=-1 in the line below also sounds strange

Maybe it is not possible to configure this via the UI, but the formatter still uses the setting?

Comment (sigiesec) [reply com.btc.serviceidl.ide/.settings/org.eclipse.xtend.core.Xtend.prefs]: I have the impression they are ignored. The indentation settings can only be set for the workspace, and are taken from the Java Editor settings. Please set them to 
* Tab policy: Spaces only
* Indentation size: 4

I will add this information to the contribution guidelines.

**Pull request merged**